racket/collects/net/http-client.rkt

```diff
@@ -121,34 +129,37 @@
                          #:headers [headers-bs empty]
                          #:content-decode [decodes '(gzip)]
                          #:data [data #f])
+  (define (print-to fmt . args)
+    ;; (eprintf "print-to: ~a~%" (apply format fmt args))
+    (apply fprintf to fmt args))
```

Comment:
I don't like this change. I don't like `apply` being used unless necessary and I don't like debugging code getting in the way of understanding and performance of normal code. Perhaps you could instead use a debugging definition of `to` that forks the output to two ports.